tests/test_plugins.py

```diff
@@ -3,29 +3,27 @@
 
 
 def load_plugin(plugin_name, ast=False):
-    _plugin = getattr(plugins, 'plugin_{}'.format(plugin_name))
+    _plugin = getattr(plugins, "plugin_{}".format(plugin_name))
```

Comment:
why do you want to change this `'` to `"`?

Reply:
I use black to refactor my code and it converts ' into ". I think black is very useful, but if it is a problem I can change.